### PR TITLE
fixed #1551

### LINKED ISSUE #1551: SemVer2 should have better prerelease package recommendations
@nkolev92 commented on [Thu Jul 18 2019](https://github.com/NuGet/Home/issues/8349)

See: 

https://docs.microsoft.com/en-us/nuget/reference/package-versioning#pre-release-versions
https://docs.microsoft.com/en-us/nuget/reference/package-versioning#semantic-versioning-200

https://docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages#semantic-versioning

In a couple of places in these articles we make a recommendation that customers should author their prerelease versions like `1.0.0-preview1`. 

When using semver2, customers should use `1.0.0-preview.1` for more predictable sorting instead of leading 0s. 

See: 

```
As shown, the version without any suffix will always take precedence over pre-release versions. Note also that if you use numerical suffixes with pre-release tags that might use double-digit numbers (or more), use leading zeroes as in beta01 and beta05 to ensure that they sort correctly when the numbers get larger.
``` 

from 
https://docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages#semantic-versioning

//cc @Mikejo5000

### PR DESCRIPTION
**Pull request by Mikejo5000**

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[44992e2](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1565/commits/44992e27a008cb939244b10b4d764918a04a3f46)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Prerelease-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-issues/docs/create-packages/Prerelease-Packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages?branch=pr-en-us-1565) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C23%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907231521061965-1565%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=gQUOovpomdsxoGyKU5ZDKANV8eI3nde3jidE2oT9%2Bns%3D&st=2019-07-23T15%3A17%3A05Z&se=2019-08-23T15%3A22%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by Mikejo5000** [comment]
@nkolev92 Nikolche, can you take a quick look? Thx.

File: docs/create-packages/Prerelease-Packages.md [hunk 1]
```diff
@@ -76,10 +76,10 @@ Whatever suffixes you use, however, NuGet will give them precedence in reverse a
     1.0.1-zzz
     1.0.1-rc
     1.0.1-open
-    1.0.1-beta12
-    1.0.1-beta05
+    1.0.1-beta.12
+    1.0.1-beta.05
```
Comment (nkolev92) [inline docs/create-packages/Prerelease-Packages.md]: The idea with semver 2 versions is that you don't need the leading 0. 

So you would have. 

`1.0.1-beta.12` and `1.0.1-beta.5`

File: docs/create-packages/Prerelease-Packages.md [hunk 2]
```diff
@@ -76,10 +76,10 @@ Whatever suffixes you use, however, NuGet will give them precedence in reverse a
...
     1.0.1-beta
-    1.0.1-alpha2
+    1.0.1-alpha.2
     1.0.1-alpha
 
-As shown, the version without any suffix will always take precedence over pre-release versions. Note also that if you use numerical suffixes with pre-release tags that might use double-digit numbers (or more), use leading zeroes as in beta01 and beta05 to ensure that they sort correctly when the numbers get larger.
+As shown, the version without any suffix will always take precedence over pre-release versions. Note also that if you use numerical suffixes with pre-release tags that might use double-digit numbers (or more), use leading zeroes as in beta.01 and beta.05 to ensure that they sort correctly when the numbers get larger.
```
Comment (nkolev92) [inline docs/create-packages/Prerelease-Packages.md]: We should suggest more explicitly that leading 0s are not needed with semver2, but they are with the old version schema.

File: docs/create-packages/Prerelease-Packages.md (hunk 1 above)
Comment (Mikejo5000) [reply docs/create-packages/Prerelease-Packages.md]: OK, thanks, updating now.

File: docs/create-packages/Prerelease-Packages.md (hunk 2 above)
Comment (Mikejo5000) [reply docs/create-packages/Prerelease-Packages.md]: Updating

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[b99880c](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1565/commits/b99880c5e46de58d0feb3a681f72afba28c93aaf)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Prerelease-Packages.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/mikejo-issues/docs/create-packages/Prerelease-Packages.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages?branch=pr-en-us-1565) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C7%5C23%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201907231854408531-1565%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=xS6DRrVWKuYqaFgpwQ9pi2G1qe%2FKdOg5TVGcvxWhD74%3D&st=2019-07-23T18%3A50%3A37Z&se=2019-08-23T18%3A55%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**